views/notes-listing.pug

```diff
@@ -21,7 +21,7 @@ block content
         | These notes are the official TC39 meeting minutes taken by&nbsp;
         a(href='http://twitter.com/rwaldron') @rwldrn
         | . They are copied from&nbsp;
-        a(href='https://github.com/rwldrn/tc39-notes/') this
+        a(href='https://github.com/rwaldron/tc39-notes/')
```

Comment:
It looks like you've accidentally removed the "this" here, so there's no actual text inside the hyperlink/anchor tag.

You should also probably update the username in `lib/notes.js` line 9 while you're at it.